### ISSUE TITLE
Jaccard Overlap for matching default boxes with ground truth boxes

### ISSUE THREAD

**Issue by atp1988** [ChunML/ssd-tf2#31]
Hello Mr. ChunML

I read your code carefully and thank you for your consideration and implementation to programming SSD with keras-tensorflow framework. I work with your code and I have a Question about Jaccard overlap for matching default boxes and ground truth boxes for better loss calculation. in the 'box_utils' file, 'compute_target' function uses Jaccard algorithms but I could not find mathematical way to implementation.
second, in 'compute_target' function you use 'encode' function which is ambiguous for me.

please help me how to write them.
thank you

Regards,
